### PR TITLE
Add How to Use This Guide Section to UG

### PR DESCRIPTION
**Pull request by maj0-0**
Have yet to update diagram in DG

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -34,14 +35,55 @@ StudentConnect is a solution for hassle-free team formation for students to brow
...
+4. [Saving the data](#saving-the-data)
+5. [Editing the data file](#editing-the-data-file)
+6. [Archiving data files [coming in v2.0]](#archiving-data-files-coming-in-v20)
+7. [FAQ](#faq)
+8. [Known issues](#known-issues)
+9. [Command summary](#command-summary)
+10. [Appendix](#appendix)
+11. [Glossary](#glossary)
 
+--------------------------------------------------------------------------------------------------------------------
```
Comment (wnchan) [inline docs/UserGuide.md:47]: thanks for adding the 'how to' guide

**Review by wnchan — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -34,14 +35,55 @@ StudentConnect is a solution for hassle-free team formation for students to brow
...
+  <img src="images/StudentListUI.png" alt="Student List View of StudentConnect" style="margin-bottom: 10px;"/>
+  <div>Figure 1: Student List View of StudentConnect</div>
+</div>
+
+<div style="text-align: center; border: 1px solid #000; padding: 10px; display: inline-block; margin: auto;">
+  <img src="images/GroupListUI.png" alt="Group List View of StudentConnect" style="margin-bottom: 10px;"/>
+  <div>Figure 2: Group List View of StudentConnect</div>
+</div>
+
+- **Command Box**: This is where you can enter any command for execution. After typing your command, press Enter to submit. If there is a syntax error or the command is unrecognizable, the text will turn red, prompting you to correct the command according to the specified requirements before pressing Enter again.
```
Comment (PearlynnT) [inline docs/UserGuide.md:72]: Use British spelling
```suggestion
- **Command Box**: This is where you can enter any command for execution. After typing your command, press Enter to submit. If there is a syntax error or the command is unrecognisable, the text will turn red, prompting you to correct the command according to the specified requirements before pressing Enter again.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -34,14 +35,55 @@ StudentConnect is a solution for hassle-free team formation for students to brow
...
+- **Student List**: The student list panel displays a roster of all students and their associated details. Executing any student-related command will result in this panel being updated to show the relevant student list.
+- **Group List**: Similar to the student list, the group list panel shows all existing groups along with their members and designated tutorials. Group-related commands will update this panel accordingly.
+- **Menu Options**: The menu options provide quick access to general tasks within the application.
+<div markdown="span" class="alert alert-primary">:exclamation: **Important:**
+To guarantee that all your changes and data are saved correctly, it is strongly advised to type the `exit` command into the Command Box
+instead of selecting `exit` from the menu options or closing the application window directly.
+This practice helps prevent data loss and ensures a proper shutdown of the application.
+</div>
+
+3. Should you encounter any terminology within the guide that is unfamiliar, you can consult the [glossary](#glossary) section below. It provides clear definitions and explanations to help you grasp the specialized language used throughout this guide.
```
Comment (PearlynnT) [inline docs/UserGuide.md:83]: Use British spelling
```suggestion
3. Should you encounter any terminology within the guide that is unfamiliar, you can consult the [glossary](#glossary) section below. It provides clear definitions and explanations to help you grasp the specialised language used throughout this guide.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -34,14 +35,55 @@ StudentConnect is a solution for hassle-free team formation for students to brow
...
+4. For any inquiries, the [FAQ](#faq) section is a good place to look for answers. In case your question is not addressed there, do not hesitate to contact us at [] for further assistance.
```
Comment (PearlynnT) [inline docs/UserGuide.md:84]: is `[]` incomplete?

Comment (maj0-0) [reply docs/UserGuide.md:84]: yep not sure what to put

Comment (wnchan) [reply docs/UserGuide.md:84]: Can put our tp github issues, then ask them to create issue to ask for assistance.

Comment (maj0-0) [reply docs/UserGuide.md]: done!

**Pull request merged**